tools/ci_connector_ops/ci_connector_ops/pipelines/README.md

```diff
@@ -146,6 +147,6 @@ A log grouping tool is under construction: https://www.youtube.com/watch&ab_chan
 
 ### Performance benchmarks
 
-| Connector      | Run integration test GHA duration | Dagger POC duration (local) | Dagger POC duration (CI) |
-|----------------|-----------------------------------|-----------------------------|--------------------------|
-| source-pokeapi | 3mn45s                            | 1mn48                       | TBD                      |
\ No newline at end of file
+| Connector      | Run integration test GHA duration                                      | Dagger POC duration (CI no cache)                                      |
+| -------------- | ---------------------------------------------------------------------- | ---------------------------------------------------------------------- |
+| source-pokeapi | [7mn22s](https://github.com/airbytehq/airbyte/actions/runs/4395453220) | [5mn26s](https://github.com/airbytehq/airbyte/actions/runs/4403595746) |
```

Comment:
🤔  There's probably a way to automate the generation of these changelogs from PR titles or conventional commits.  We make these in so many sub-projects